src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPrintMembers.cs

```diff
@@ -207,6 +206,27 @@ static BoundStatement makeAppendString(SyntheticBoundNodeFactory F, BoundParamet
             {
                 return F.ExpressionStatement(F.Call(receiver: builder, F.WellKnownMethod(WellKnownMember.System_Text_StringBuilder__AppendString), F.StringLiteral(value)));
             }
+
+            static bool isPrintable(Symbol m)
+            {
+                if (m.DeclaredAccessibility != Accessibility.Public)
+                {
+                    return false;
+                }
+
+                if (m.Kind is SymbolKind.Field)
+                {
+                    return true;
+                }
+
+                if (m.Kind is SymbolKind.Property)
```

Comment:
I feel like there are more fun uses of patterns possible here 😉 but as it doesn't affect correctness or readability I would not ask you to change anything.